### DISCUSSION TITLE
TRPC with prisma and clerk

### DISCUSSION THREAD

**Discussion by hrutik7** [trpc/trpc#5265] [Q&A]
I am using prisma and clerk along with the TRPC . and I want to build protected routes but after follwing some of the docs + blogs I am getting errors here is the repo : - https://github.com/hrutik7/Emma 
i am using monorepo for my project so I am clear about t3 stack but question is how I can integrate the same in monorepo 
thanks

**Comment by johannespn** [comment]
You check this https://www.jumr.dev/blog/t3-turbo
